creator-node/src/services/sync/secondarySyncFromPrimary.ts

```diff
@@ -845,6 +858,30 @@ async function _secondarySyncFromPrimary({
   return { result }
 }
 
+const wasThisNodeEverPrimaryFor = async (
+  wallet: string,
+  libs: any
+): Promise<boolean> => {
+  // Get userId from wallet
+  const users = await libs.User.getUsers(1, 0, null, wallet)
+  const userId = users[0].user_id
+
+  // Return true if any snapshot of the user's history had this node as their primary
+  const history = (
+    await axios({
+      baseURL: libs.discoveryProvider.discoveryProviderEndpoint,
+      url: `/users/history/${userId}`,
```

Comment:
by default this only returns 100, we can bump to 500 by requesting 
```suggestion
      url: `users/history/${userId}?limit=500`,
```